src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_PaymentOptionsScreen.tsx

```diff
@@ -71,7 +71,7 @@ export const Profile_PaymentOptionsScreen = ({
           if (responseCode === 'OK') {
             await authorizeRecurringPayment(paymentId);
             await refreshCards();
-          } else if (responseCode === 'CANCEL') {
```

Comment:
This was probably tested at some point, and worked. If the response code can actually change capitalisation, we should rather use case-insensitive comparison.

Reply:
I'll try to convert everything to lowercase before comparison, thanks for the pointer.